### DISCUSSION TITLE
Εβδομάδα 7 και άσκηση γραμμής εντολών

### DISCUSSION THREAD

**Discussion by p14kairas** [courses-ionio/hci#1645] [Announcements]
Θα μπορούσαμε να κάνουμε μία warm-up άσκηση γραμμής εντολών ως παραδοτέο για την εβδομάδα 7 (μέχρι σήμερα δηλαδη) καθώς υπάρχει και η εβδομάδα 9 και 11 που έχουν άσκηση γραμμής εντολών ή θέλετε να είναι από την κατηγορία HCI η μπορούμε να διαλέξουμε εμέις ποια εβδομάδα θα κάνουμε τι.?

**Comment by Sola1r3** [comment]
Νομίζω ότι οι warmup ασκήσεις αναφέρονται παραπάνω στα παραδοτέα των πρώτων 6 εβδομάδων. Όποτε ίσως να μπορείς να τις προσθέσεις σε αυτά που αναφέρονται στην εγακτάσταση ArchLinux και στην εξοικείωση σου με αυτό. Η 7, η 9 και η 11 αναφέρονται στις δύο κύριες ασκήσεις γραμμής εντολών με την μία έξτρα εβδομάδα να είναι σαν μπόνους για να τα δουλέψεις παραπάνω.
